hack/dockerfile/install/tini.installer

```diff
@@ -1,12 +1,13 @@
 #!/bin/sh
 
-: ${TINI_COMMIT:=fec3683b971d9c3ef73f284f176672c44b448662} # v0.18.0
```

Comment:
Note; we likely need a make target, or alternative script, because I just discovered that the Docker packaging scripts rely on these; https://github.com/docker/docker-ce-packaging/blob/09b3ac8/deb/common/rules#L12-L15

```
override_dh_auto_build:
	# Build the daemon and dependencies
	cd engine && PRODUCT=docker ./hack/make.sh dynbinary
	cd engine && TMP_GOPATH="/go" hack/dockerfile/install/install.sh tini
	cd engine && TMP_GOPATH="/go" hack/dockerfile/install/install.sh proxy dynamic
	# Build the CLI
```